src/containers/userInfo.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+let state = {
+  email: ""
+};
+// const api_endpoint = "https://sei19-safira.herokuapp.com/api/v1/user"
```

Comment:
Can we remove this comment? Hardcoded URL

Reply:
im gonna do it